packages/generator-js/src/base/index.ts

```diff
@@ -0,0 +1,100 @@
+import Generator from 'yeoman-generator';
+
+import { ConfigureGenerator } from '../app';
+
+export default class extends ConfigureGenerator {
+  initializing() {
+    super.initializing();
+    // default until testing is set
+    this.config.set('testing', false);
+  }
+
+  async prompting() {
+    const prompts: Array<Generator.Question> = [
+      {
+        type: 'checkbox',
+        name: 'features',
+        message: 'What features would you like to include?',
+        choices: ['storybook', 'testing', 'CI'],
+        default: ['testing', 'CI'],
```

Comment:
Maybe change these options to CircleCI since the config is specific to Circle

Reply:
may have options for travis in the future and that would be split after selecting this